src/storage/mod.rs

```diff
@@ -401,6 +473,19 @@ where
             data: &mut self.data,
         }
     }
+
+    /*
+    pub fn meta_mut(&mut self) -> &mut T::Metadata {
```

Comment:
any reason this is commented out?

Reply:
Oop, that was a mistake, I wanted to make sure I changed most of the documentation to use `find::<T>();`.